src/libraries/System.ServiceProcess.ServiceController/tests/ServiceBaseTests.cs

```diff
@@ -136,6 +136,7 @@ public void TestOnPauseAndContinueThenStop()
             controller.WaitForStatus(ServiceControllerStatus.Stopped);
         }
 
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/1724")]
```

Comment:
Used the URL form of the issue vs. simple `int` to avoid confusion with `int` that refer to corefx issue ids. Happy to switch to `int` form if preferred.